src/api/deploy/aws-code-deploy.ts

```diff
@@ -142,16 +142,16 @@ export function codeDeploy(
                 .toPromise()
                 .then(result => {
                   if (!(result && result.data === 0)) {
-                    const msg = 'create-deployment-group failed';
-                    observer.next({ type: 'containerError', data: msg});
+                    const m = 'create-deployment-group failed';
```

Comment:
you had `msg` previously, then you use `message` sometimes and here you have `m` 

Please consolidate and pick one descriptive name.

Reply:
they were shadowed variables, I just wanted to make tslint pass.